core/src/test/java/org/apache/calcite/rex/RexBuilderTest.java

```diff
@@ -960,6 +960,22 @@ private void checkDate(RexLiteral literal) {
     assertThat(inCall.getKind(), is(SqlKind.SEARCH));
   }
 
+  @Test void testMakeInWithArrayLiterals() {
```

Comment:
Do we need to add a Jira link on this method?

Reply:
Fixed in https://github.com/apache/calcite/pull/4345/commits/280190b4be82f8fdb4c87096eed3ddb9d6ed863d